requirements.txt

```diff
@@ -0,0 +1,5 @@
+requests
+colorama
+win_unicode_console
+pysnmp-lextudio==5.0.31
+pyasn1==0.6.0
```

Comment:
This particular version requirements and one above was the key insight that made discovery work for me. Thank you!